src/main/java/racing/domain/Car.java

```diff
@@ -1,31 +1,69 @@
 package racing.domain;
 
+import java.util.Objects;
+
 public class Car {
     private static final int DEFAULT_POSITION = 1;
+    private String name;
     private int position;
 
     public Car() {
         position = DEFAULT_POSITION;
     }
 
-    public Car setPosition(int inputPosition) {
-        position = inputPosition;
-        return this;
+    public Car(String name) {
+        this.name  = name;
+        this.position = DEFAULT_POSITION;
```

Comment:
자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
        this(name, DEFAULT_POSITION);
```

Reply:
앗 강의 때 생성자는 parameter가 적은 생성자가 많은 생성자를 호출하도록 만들어 주는게 좋다는 내용을 들었는데 적용을 못했었네요!!
적용 해 봐야겠어요 :) 피드백 감사합니다!!